resources/views/settings/index.blade.php

```diff
@@ -444,10 +444,12 @@
                 }
             });
 
-            $('.voyager-trash').click(function () {
+            $('.panel-actions .voyager-trash').click(function () {
```

Comment:
What's this class for?

Reply:
This fixes the error when a user clicked the trash icon in the modal the value went: undefined/undefined. This ensures that only the trash icons in the actions are used to get the data-id.